<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Управление</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton_disabled_mode">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Выключен</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_static_mode">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Задание положения</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_sine_mode">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Синусоида</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Режим</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_des_val">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Задание положения</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Set_angle">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>W</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>221</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Калибровка</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>172</width>
          <height>73</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Kp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="Kp_text"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Set_angle_2">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>130</y>
          <width>171</width>
          <height>73</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ki</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="Ki_text"/>
         </item>
         <item>
          <widget class="QPushButton" name="write_Ki">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>220</y>
          <width>172</width>
          <height>73</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Kd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="Kd_text"/>
         </item>
         <item>
          <widget class="QPushButton" name="write_Kd">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
